### ISSUE TITLE
[feature request] Indeterminated state, start point

### ISSUE THREAD

**Issue by drakulis** [wyzantinc/vue-radial-progress#42]
First of all thanks for this awesome component. Works perfectly!

I would like to propose two additional features which could improve it a bit.
1. Indeterminated state - it could be some animation where some part of the circle (lets say 10%) is constantly rotating
2. Start point - currently value arc is started from the top of the circle. Would be awesome to be able to set it - could be set as a degree. This way I will also be able to create indeterminated state spinner - driven by value of the degree.

Thanks!

**Comment by signalwerk** [comment]
@drakulis I like your second Idea. Do you have a PR for it?

**Comment by drakulis** [comment]
@signalwerk No, I don't. It just comes to my mind when I was using this component.

**Comment by setaman** [comment]
@drakulis I was looking for these features a while ago, but unfortunately I didn't find them. So I wrote my own library that has these and other features requested in this and other projects out of the box. Just for the case that it could be useful for you too.
